lib/backend/report.go

```diff
@@ -142,6 +148,11 @@ func (s *Reporter) GetRange(ctx context.Context, startKey []byte, endKey []byte,
 		reads.WithLabelValues(s.Component).Add(float64(len(res.Items)))
 	}
 	s.trackRequest(types.OpGet, startKey, endKey)
+	if d := s.Clock().Since(start); d > time.Second*3 {
+		if s.slowRangeLogLimiter.Allow() {
```

Comment:
Technically...

```suggestion
	end := s.Clock().Now()
	if d := end.Since(start); d > time.Second*3 {
		if s.slowRangeLogLimiter.AllowN(end, 1) {
```